src/sentry/notifications/utils/__init__.py

```diff
@@ -335,19 +337,27 @@ def get_span_evidence_value(span: Optional[Dict[str, Union[str, float, None]]] =
     return render_to_string("sentry/emails/transactions.html", context)
 
 
+def get_matched_problem(event: Event) -> EventPerformanceProblem:
```

Comment:
The return value should be `Optional`, and we'll need to handle the case where this returns None